### ISSUE TITLE
mac 运行没有结果，没有有效日志。

### ISSUE THREAD

**Issue by xydz** [PaddlePaddle/PaddleOCR#15727]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

```python 
from pathlib import Path
from paddleocr import PPStructureV3


img_path = Path("examples") / "crm" / "error" / "App-1793.jpg"

if img_path.exists():
    pipeline = PPStructureV3(
        use_doc_orientation_classify=False,
        use_doc_unwarping=False,
        enable_mkldnn=False
    )
    print("--------------------------------start--------------------------------")
    # For Image
    output = pipeline.predict(input=str(img_path))

    print(f"预测完成，输出类型: {type(output)}")
    print(f"输出长度: {len(output) if hasattr(output, '__len__') else 'N/A'}")
    print("输出内容:")
    print(output)

    # 可视化结果并保存 json 结果
    for res in output:
        res.print(res) 
        res.save_to_json(save_path="output") 
        res.save_to_markdown(save_path="output") 

    print("done")
else:
    print(f"图片不存在: {img_path.resolve()}")
```
运行结果：
“uv run python ./src/2025.py
Creating model: ('PP-DocBlockLayout', None)
Using official model (PP-DocBlockLayout), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
~/code/AiProjects/pdf/paddle/.venv/lib/python3.11/site-packages/paddle/utils/cpp_extension/extension_utils.py:711: UserWarning: No ccache found. Please be aware that recompiling all source files may be required. You can download and install ccache from: https://github.com/ccache/ccache/blob/master/doc/INSTALL.md
  warnings.warn(warning_message)
Creating model: ('PP-DocLayout_plus-L', None)
Using official model (PP-DocLayout_plus-L), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Creating model: ('PP-LCNet_x1_0_textline_ori', None)
Using official model (PP-LCNet_x1_0_textline_ori), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Creating model: ('PP-OCRv5_server_det', None)
Using official model (PP-OCRv5_server_det), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Creating model: ('PP-OCRv5_server_rec', None)
Using official model (PP-OCRv5_server_rec), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Creating model: ('PP-OCRv4_server_seal_det', None)
Using official model (PP-OCRv4_server_seal_det), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Creating model: ('PP-OCRv5_server_rec', None)
Using official model (PP-OCRv5_server_rec), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Creating model: ('PP-LCNet_x1_0_table_cls', None)
Using official model (PP-LCNet_x1_0_table_cls), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Creating model: ('SLANeXt_wired', None)
Using official model (SLANeXt_wired), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Creating model: ('SLANet_plus', None)
Using official model (SLANet_plus), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Creating model: ('RT-DETR-L_wired_table_cell_det', None)
Using official model (RT-DETR-L_wired_table_cell_det), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Creating model: ('RT-DETR-L_wireless_table_cell_det', None)
Using official model (RT-DETR-L_wireless_table_cell_det), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Creating model: ('PP-FormulaNet_plus-L', None)
Using official model (PP-FormulaNet_plus-L), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Creating model: ('PP-Chart2Table', None)
Using official model (PP-Chart2Table), the model files will be automatically downloaded and saved in ~/.paddlex/official_models.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Loading configuration file ~/.paddlex/official_models/PP-Chart2Table/config.json
Loading weights file ~/.paddlex/official_models/PP-Chart2Table/model_state.pdparams
Loaded weights file from disk, setting weights to model.
All model checkpoint weights were used when initializing PPChart2TableInference.

All the weights of PPChart2TableInference were initialized from the model checkpoint at ~/.paddlex/official_models/PP-Chart2Table.
If your task is similar to the task the model of the checkpoint was trained on, you can already use PPChart2TableInference for predictions without further training.
Loading configuration file ~/.paddlex/official_models/PP-Chart2Table/generation_config.json
--------------------------------start--------------------------------
”

### 🏃‍♂️ Environment (运行环境)

```bash
OS : macOS-13.5.2
Environment : Jupyter
Python : 3.11.3
PaddleOCR : 3.0.1
Install : uv
RAM : 16.00 GB
CPU : Apple M3
CUDA: None
```

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

```python 
from pathlib import Path
from paddleocr import PPStructureV3


img_path = Path("examples") / "crm" / "error" / "App-1793.jpg"

if img_path.exists():
    pipeline = PPStructureV3(
        use_doc_orientation_classify=False,
        use_doc_unwarping=False,
        enable_mkldnn=False
    )
    print("--------------------------------start--------------------------------")
    # For Image
    output = pipeline.predict(input=str(img_path))

    print(f"预测完成，输出类型: {type(output)}")
    print(f"输出长度: {len(output) if hasattr(output, '__len__') else 'N/A'}")
    print("输出内容:")
    print(output)

    # 可视化结果并保存 json 结果
    for res in output:
        res.print(res) 
        res.save_to_json(save_path="output") 
        res.save_to_markdown(save_path="output") 

    print("done")
else:
    print(f"图片不存在: {img_path.resolve()}")

**Comment by yandezhi15** [comment]
补充说明 
uv tree
Resolved 101 packages in 6ms
paddle-project v0.1.0
├── paddleocr v3.0.1
│   ├── paddlex[ie, multimodal, ocr] v3.0.1
│   │   ├── chardet v5.2.0
│   │   ├── colorlog v6.9.0
│   │   ├── filelock v3.18.0
│   │   ├── gputil v1.4.0
│   │   ├── numpy v1.24.4
│   │   ├── packaging v24.2
│   │   ├── pandas v1.5.3
│   │   │   ├── numpy v1.24.4
│   │   │   ├── python-dateutil v2.9.0.post0
│   │   │   │   └── six v1.17.0
│   │   │   └── pytz v2025.2
│   │   ├── pillow v11.2.1
│   │   ├── prettytable v3.16.0
│   │   │   └── wcwidth v0.2.13
│   │   ├── py-cpuinfo v9.0.0
│   │   ├── pydantic v2.11.6
│   │   │   ├── annotated-types v0.7.0
│   │   │   ├── pydantic-core v2.33.2
│   │   │   │   └── typing-extensions v4.14.0
│   │   │   ├── typing-extensions v4.14.0
│   │   │   └── typing-inspection v0.4.1
│   │   │       └── typing-extensions v4.14.0
│   │   ├── pyyaml v6.0.2
│   │   ├── requests v2.32.4
│   │   │   ├── certifi v2025.4.26
│   │   │   ├── charset-normalizer v3.4.2
│   │   │   ├── idna v3.10
│   │   │   └── urllib3 v2.4.0
│   │   ├── ruamel-yaml v0.18.14
│   │   │   └── ruamel-yaml-clib v0.2.12
│   │   ├── typing-extensions v4.14.0
│   │   ├── ujson v5.10.0
│   │   ├── ftfy v6.3.1 (extra: ie)
│   │   │   └── wcwidth v0.2.13
│   │   ├── imagesize v1.4.1 (extra: ie)
│   │   ├── langchain v0.2.17 (extra: ie)
│   │   │   ├── aiohttp v3.12.12
│   │   │   │   ├── aiohappyeyeballs v2.6.1
│   │   │   │   ├── aiosignal v1.3.2
│   │   │   │   │   └── frozenlist v1.7.0
│   │   │   │   ├── attrs v25.3.0
│   │   │   │   ├── frozenlist v1.7.0
│   │   │   │   ├── multidict v6.4.4
│   │   │   │   ├── propcache v0.3.2
│   │   │   │   └── yarl v1.20.1
│   │   │   │       ├── idna v3.10
│   │   │   │       ├── multidict v6.4.4
│   │   │   │       └── propcache v0.3.2
│   │   │   ├── langchain-core v0.2.43
│   │   │   │   ├── jsonpatch v1.33
│   │   │   │   │   └── jsonpointer v3.0.0
│   │   │   │   ├── langsmith v0.1.147
│   │   │   │   │   ├── httpx v0.28.1
│   │   │   │   │   │   ├── anyio v4.9.0
│   │   │   │   │   │   │   ├── idna v3.10
│   │   │   │   │   │   │   ├── sniffio v1.3.1
│   │   │   │   │   │   │   └── typing-extensions v4.14.0
│   │   │   │   │   │   ├── certifi v2025.4.26
│   │   │   │   │   │   ├── httpcore v1.0.9
│   │   │   │   │   │   │   ├── certifi v2025.4.26
│   │   │   │   │   │   │   └── h11 v0.16.0
│   │   │   │   │   │   └── idna v3.10
│   │   │   │   │   ├── orjson v3.10.18
│   │   │   │   │   ├── pydantic v2.11.6 (*)
│   │   │   │   │   ├── requests v2.32.4 (*)
│   │   │   │   │   └── requests-toolbelt v1.0.0
│   │   │   │   │       └── requests v2.32.4 (*)
│   │   │   │   ├── packaging v24.2
│   │   │   │   ├── pydantic v2.11.6 (*)
│   │   │   │   ├── pyyaml v6.0.2
│   │   │   │   ├── tenacity v8.5.0
│   │   │   │   └── typing-extensions v4.14.0
│   │   │   ├── langchain-text-splitters v0.2.4
│   │   │   │   └── langchain-core v0.2.43 (*)
│   │   │   ├── langsmith v0.1.147 (*)
│   │   │   ├── numpy v1.24.4
│   │   │   ├── pydantic v2.11.6 (*)
│   │   │   ├── pyyaml v6.0.2
│   │   │   ├── requests v2.32.4 (*)
│   │   │   ├── sqlalchemy v2.0.41
│   │   │   │   └── typing-extensions v4.14.0
│   │   │   └── tenacity v8.5.0
│   │   ├── langchain-community v0.2.17 (extra: ie)
│   │   │   ├── aiohttp v3.12.12 (*)
│   │   │   ├── dataclasses-json v0.6.7
│   │   │   │   ├── marshmallow v3.26.1
│   │   │   │   │   └── packaging v24.2
│   │   │   │   └── typing-inspect v0.9.0
│   │   │   │       ├── mypy-extensions v1.1.0
│   │   │   │       └── typing-extensions v4.14.0
│   │   │   ├── langchain v0.2.17 (*)
│   │   │   ├── langchain-core v0.2.43 (*)
│   │   │   ├── langsmith v0.1.147 (*)
│   │   │   ├── numpy v1.24.4
│   │   │   ├── pyyaml v6.0.2
│   │   │   ├── requests v2.32.4 (*)
│   │   │   ├── sqlalchemy v2.0.41 (*)
│   │   │   └── tenacity v8.5.0
│   │   ├── langchain-core v0.2.43 (extra: ie) (*)
│   │   ├── langchain-openai v0.1.25 (extra: ie)
│   │   │   ├── langchain-core v0.2.43 (*)
│   │   │   ├── openai v1.63.2
│   │   │   │   ├── anyio v4.9.0 (*)
│   │   │   │   ├── distro v1.9.0
│   │   │   │   ├── httpx v0.28.1 (*)
│   │   │   │   ├── jiter v0.10.0
│   │   │   │   ├── pydantic v2.11.6 (*)
│   │   │   │   ├── sniffio v1.3.1
│   │   │   │   ├── tqdm v4.67.1
│   │   │   │   └── typing-extensions v4.14.0
│   │   │   └── tiktoken v0.9.0
│   │   │       ├── regex v2024.11.6
│   │   │       └── requests v2.32.4 (*)
│   │   ├── lxml v5.4.0 (extra: ie)
│   │   ├── openai v1.63.2 (extra: ie) (*)
│   │   ├── opencv-contrib-python v4.10.0.84 (extra: ie)
│   │   │   └── numpy v1.24.4
│   │   ├── openpyxl v3.1.5 (extra: ie)
│   │   │   └── et-xmlfile v2.0.0
│   │   ├── premailer v3.10.0 (extra: ie)
│   │   │   ├── cachetools v6.0.0
│   │   │   ├── cssselect v1.3.0
│   │   │   ├── cssutils v2.11.1
│   │   │   │   └── more-itertools v10.7.0
│   │   │   ├── lxml v5.4.0
│   │   │   └── requests v2.32.4 (*)
│   │   ├── pyclipper v1.3.0.post6 (extra: ie)
│   │   ├── pypdfium2 v4.30.1 (extra: ie)
│   │   ├── scikit-learn v1.7.0 (extra: ie)
│   │   │   ├── joblib v1.5.1
│   │   │   ├── numpy v1.24.4
│   │   │   ├── scipy v1.15.3
│   │   │   │   └── numpy v1.24.4
│   │   │   └── threadpoolctl v3.6.0
│   │   ├── shapely v2.1.1 (extra: ie)
│   │   │   └── numpy v1.24.4
│   │   ├── tokenizers v0.19.1 (extra: ie)
│   │   │   └── huggingface-hub v0.33.0
│   │   │       ├── filelock v3.18.0
│   │   │       ├── fsspec v2025.5.1
│   │   │       ├── hf-xet v1.1.3
│   │   │       ├── packaging v24.2
│   │   │       ├── pyyaml v6.0.2
│   │   │       ├── requests v2.32.4 (*)
│   │   │       ├── tqdm v4.67.1
│   │   │       └── typing-extensions v4.14.0
│   │   ├── einops v0.8.1 (extra: multimodal)
│   │   ├── ftfy v6.3.1 (extra: multimodal) (*)
│   │   ├── jinja2 v3.1.6 (extra: multimodal)
│   │   │   └── markupsafe v3.0.2
│   │   ├── opencv-contrib-python v4.10.0.84 (extra: multimodal) (*)
│   │   ├── pypdfium2 v4.30.1 (extra: multimodal)
│   │   ├── regex v2024.11.6 (extra: multimodal)
│   │   ├── tiktoken v0.9.0 (extra: multimodal) (*)
│   │   ├── ftfy v6.3.1 (extra: ocr) (*)
│   │   ├── imagesize v1.4.1 (extra: ocr)
│   │   ├── lxml v5.4.0 (extra: ocr)
│   │   ├── opencv-contrib-python v4.10.0.84 (extra: ocr) (*)
│   │   ├── openpyxl v3.1.5 (extra: ocr) (*)
│   │   ├── premailer v3.10.0 (extra: ocr) (*)
│   │   ├── pyclipper v1.3.0.post6 (extra: ocr)
│   │   ├── pypdfium2 v4.30.1 (extra: ocr)
│   │   ├── scikit-learn v1.7.0 (extra: ocr) (*)
│   │   ├── shapely v2.1.1 (extra: ocr) (*)
│   │   └── tokenizers v0.19.1 (extra: ocr) (*)
│   ├── pyyaml v6.0.2
│   └── typing-extensions v4.14.0
├── setuptools v80.9.0
├── paddlepaddle v3.0.0 (extra: cpu)
│   ├── astor v0.8.1
│   ├── decorator v5.2.1
│   ├── httpx v0.28.1 (*)
│   ├── networkx v3.5
│   ├── numpy v1.24.4
│   ├── opt-einsum v3.3.0
│   │   └── numpy v1.24.4
│   ├── pillow v11.2.1
│   ├── protobuf v3.20.2
│   └── typing-extensions v4.14.0
└── paddlepaddle-gpu v2.6.2 (extra: gpu)
    ├── astor v0.8.1
    ├── decorator v5.2.1
    ├── httpx v0.28.1 (*)
    ├── numpy v1.24.4
    ├── opt-einsum v3.3.0 (*)
    ├── pillow v11.2.1
    └── protobuf v3.20.2
(*) Package tree already displayed

**Comment by tom123478** [comment]
目测mac的 CPU 一次性加载多个子模型太慢了，默认只输出 ERROR 及以上日志，没有任何输出。建议将日志级别调到 INFO，重新运行后观察各个模型加载和推理的进度

**Comment by cuicheng01** [comment]
请问现在的问题解决了吗？

**Comment by GreatV** [comment]
uv好像还不支持paddle框架

**Comment by cuicheng01** [comment]
哦，注意到使用的是uv，目前uv确实不支持paddle框架

**Comment by GreatV** [comment]
也有可能是内存不足，可以试试mobile版本的模型。
